_examples/README.md

```diff
@@ -2,6 +2,31 @@
 
 > All code snippets are run from the root project directory.
 
+## Run Waku node
+
+Running Waku node is a matter of a correct configuration. To enable Waku and JSON-RPC HTTP interface use:
+```shell script
+{
+  "APIModules": "waku",
+  "HTTPEnabled": true,
+  "HTTPHost": "localhost",
+  "HTTPPort": "8545",
+  "WakuConfig": {
+    "Enabled": true
+  }
+}
+```
+
+This command will start a Waku node using the `eth.prod` fleet:
```

Comment:
The default waku.json there does not seem to open HTTP RPC endpoint. Only ipc. Looks like that only works if you also provide HTTPHost.
Perhaps make it clear that you should use the new config and not the one of the examples.

Reply:
> Running Waku node is a matter of a correct configuration. To enable Waku and JSON-RPC HTTP interface use

I think this is pretty clear in the context of this README.

But I can also update `*.json` files to enable HTTP.

Reply:
Done